imgui/CMakeLists.txt

```diff
@@ -69,15 +69,27 @@ endif()
 file(GLOB imgui_sources ${imgui_SOURCE_DIR}/*.cpp ${imgui_SOURCE_DIR}/misc/cpp/*.cpp)
 file(GLOB implot_sources ${implot_SOURCE_DIR}/*.cpp)
 file(GLOB fonts_sources ${fonts_SOURCE_DIR}/src/*.cpp)
-add_library(imgui STATIC
-    ${imgui_sources}
-    ${implot_sources}
-    ${imgui_SOURCE_DIR}/backends/imgui_impl_glfw.cpp
-    ${imgui_SOURCE_DIR}/backends/imgui_impl_opengl3.cpp
-    ${gl3w_BINARY_DIR}/src/gl3w.c
-    ${fonts_sources}
-    src/stb_image.cpp
-)
+if (MSVC)
+  add_library(imgui STATIC
+      ${imgui_sources}
+      ${implot_sources}
+      ${imgui_SOURCE_DIR}/backends/imgui_impl_glfw.cpp
+      ${imgui_SOURCE_DIR}/backends/imgui_impl_opengl3.cpp
+      ${gl3w_BINARY_DIR}/src/gl3w.c
+      ${fonts_sources}
+      src/stb_image.cpp
+  )
+else()
+  add_library(imgui
+      ${imgui_sources}
+      ${implot_sources}
+      ${imgui_SOURCE_DIR}/backends/imgui_impl_glfw.cpp
+      ${imgui_SOURCE_DIR}/backends/imgui_impl_opengl3.cpp
+      ${gl3w_BINARY_DIR}/src/gl3w.c
+      ${fonts_sources}
+      src/stb_image.cpp
+  )
+endif()
```

Comment:
To avoid impacting two lists if this ever changes, create a imgui_all_sources variable so the add_library becomes a one liner.